contracts/governance/BaseWeightedMultisig.sol

```diff
@@ -200,8 +200,6 @@ abstract contract BaseWeightedMultisig is IBaseWeightedMultisig {
         uint256 signerIndex;
         uint256 totalWeight;
 
-        if (signaturesLength == 0) revert LowSignaturesWeight();
```

Comment:
confirm that our unit tests cover this scenario

Reply:
Yup, Confirmed that `BaseWeightedMultisig` has 100% coverage for `% Stmts | % Branch |  % Funcs |  % Lines` and `signaturesLength == 0` case was covered, also `revert LowSignaturesWeight()` is called `3 times`.